File: code/client/src/enumerates/_enum.js
/*%@ return data.dataModel.enums.map(function(en) {
  return {
    fileName: normalize(en.name.toLowerCase()) + '.js',
    context: en
  };
})
%*/
export default [
/*%
  context.values.forEach(function(val) {
%*/
  { text: "/*%= normalize(context.name.toLowerCase()) + '.' + val %*/", value: "/*%= val %*/"},
/*%
  });
%*/
];

File: code/server/src/main/java/es/udc/lbd/gema/lps/model/domain/_enum.java
/*%@ return data.dataModel.enums.map(function(en) {
    return {
        fileName: normalize(en.name, true) + '.java',
        context: en
    };
}) %*/
package es.udc.lbd.gema.lps.model.domain;

public enum /*%= normalize(context.name, true) %*/ {
    /*%= context.values.join(', ') %*/
}
